clean-dataset/clean-data/library.whizzml

```diff
@@ -0,0 +1,113 @@
+;; extract-meta
+;;
+;; Accepts a nested map for one field in the ":fields" nested map in
+;; the dataset meta-data and returns a filtered map
+;;
+;; Inputs:
+;;   mpi: (map) Nested for one field in the ":fields" nested map
+;;
+;; Output: (map) Filtered map
+;;
+(define (extract-meta mpi) 
+  (let (mpis (get mpi "summary")
+        mpos (assoc {} "mean" (get mpis "mean")
+                       "median" (get mpis "median")
+                       "minimum" (get mpis "minimum")
+                       "maximum" (get mpis "maximum")))
+    (assoc {} "datatype" (get mpi "datatype") 
+              "name" (get mpi "name")
+              "optype" (get mpi "optype")
+              "summary" mpos)))
```

Comment:
instead of `(assoc {} "foo" bar "baz" quux)` on can simply write `{"foo" bar "baz" quux}`